app/models/post_like.rb

```diff
@@ -0,0 +1,4 @@
+class PostLike < ApplicationRecord
+  belongs_to :user, dependent: :destroy
```

Comment:
I am not sure about this step. If we do "user, dependent: :destroy" we are saying that when you destroy a PostLike also destroy the user. See point 4.1.2.4 :dependent from https://guides.rubyonrails.org/association_basics.html